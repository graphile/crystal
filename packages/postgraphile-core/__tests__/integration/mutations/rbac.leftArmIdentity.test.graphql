#> rbac: true
#> options: { ignoreRBAC: false }

mutation {
  leftArmIdentity(input:{
    leftArm:{
      # Note we can specify all the fields here because it uses the base type
      id: 9001 # it's over 9000!
      personId: 99
      lengthInMetres: 77
      mood: "jubilant"
    }
  }) {
    leftArm {
      nodeId
      id
      personId
      lengthInMetres
      mood
    }
  }
}
