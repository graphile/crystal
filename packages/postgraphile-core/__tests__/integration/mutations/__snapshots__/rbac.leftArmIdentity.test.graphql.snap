// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`execute 1`] = `
Object {
  "data": Object {
    "leftArmIdentity": Object {
      "leftArm": Object {
        "id": 9001,
        "lengthInMetres": 77,
        "mood": "jubilant",
        "nodeId": "WyJsZWZ0X2FybXMiLDkwMDFd",
        "personId": 99,
      },
    },
  },
}
`;
